File: src/class/util/shape.py
from classes.util.coordinate import Coordinate
from typing import List


class Shape:
    def __init__(self, coordinates: List[Coordinate]):
        self.coordinates = coordinates

    def __str__(self):
        i = 0
        return_value = "{"
        return_value += "coordinates:["
        for coordinate in self.coordinates:
            if i != 0:
                return_value += ","
            return_value += str(coordinate)
            i += 1
        return return_value + "}"

File: src/class/util/shape.ts
import { Coordinate } from "./coordinate"
export class Shape{
    coordinates: Coordinate[];
    constructor(coordinates:Coordinate[]){
        this.coordinates = coordinates
    }
    public toString(){
        let i = 0;
        let return_value = "{";
        return_value += "coordinates:[";
        for(let coordinate in this.coordinates){
            if(i != 0){
                return_value += ","
            }
            return_value += coordinate.toString()
            i += 1
        }
        return_value += "}";
        return  return_value;
    }
}